sentences/en/_common.yaml

```diff
@@ -285,12 +285,18 @@ lists:
         out: "closed"
 
   # sensors
-  sensor_classes:
+  sensor_temperature:
```

Comment:
I can't understand why is this changed. My idea is to create logic like covers have - there is cover devices. That way fewer sentences are needed.

Reply:
Because you need sentences per device class, like for the binary sensors.  I could not get it working otherwise.

Reply:
But it could be I'm wrong here. But this is how I got a working solution.

Reply:
I was trying to create it like in covers ("(is|are) any {cover_classes:device_class} {cover_states:state} [in <area>]"). I can't figure out why that approach isn't working..